tjreports/site/models/reports.php

```diff
@@ -326,7 +326,7 @@ protected function getListQuery()
 			foreach ($displayFilter as $key => $dispFilter)
 			{
 				// Check if any of the filter is set
-				if ((($filters[$key] != '') || substr($dispFilter['search_type'], -6) === '.range') && in_array($key, $colToshow))
```

Comment:
Please read this - 
https://www.virendrachandak.com/techtalk/php-isset-vs-empty-vs-is_null/

Reply:
@coolbung I am accepting this for now as I cannot see an alternative. We were having empty condition which was not working if filter has 0(zero) in it.